.github/workflows/lint-and-test.yml

```diff
@@ -14,5 +14,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm install
+      - run: npm run version:output
```

Comment:
Probably need to run https://github.com/journy-io/js-sdk/blob/main/.github/workflows/build-and-release.yml#L22 before this?

Reply:
Ah no, this is for the build and release workflow :)

Reply:
We still need to do this for build and release, first we need to set the package.json version using https://github.com/journy-io/js-sdk/blob/main/.github/workflows/build-and-release.yml#L22

then trigger the build...

Reply:
Normally, this is already included in the build, right? https://github.com/journy-io/js-sdk/blob/user-agent-version/package.json#L21

Reply:
Ah, yeah, it already happens before the build, my bad... 🍨